packages/gatsby-theme-aio/src/components/Hero/index.js

```diff
@@ -27,11 +27,13 @@ import {
   rootFixPages,
   rootFix,
   cleanRootFix,
-  findSelectedTopPageMenu
+  findSelectedTopPageMenu,
+  getCleanBoolean
 } from '../../utils';
 import Context from '../Context';
 import { Breadcrumbs } from '../Breadcrumbs';
 import classNames from 'classnames';
+import { utils } from 'axe-core';
```

Comment:
Why do you need `axe-core` utils here ?

Reply:
i don't.  That's a mistake.

Reply:
done.  dumb auto import has been removed